### PR TITLE
Add Javadoc for Calculator.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/finitestatemachine/TransitionMatrix.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import java.util.Set;
 
 /**
- * An interface to describe a transition matrix.
+ * An interface to  describe a transition matrix.
  *
  * @param <State> Possible states
  * @author Oleg Melnik
```
Comment (alexander-yevsyukov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/finitestatemachine/TransitionMatrix.java]: > @param <State> Possible states
Assuming we're describing a generic type parameter, it's rather “a type of possible states”

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java [hunk 2]
```diff
@@ -35,14 +35,14 @@
...
      */
     public FSMCalculator() {
         super(new CalculatorTransitions(), new ParsingService());
     }
 
     /**
-     * Evaluates math expression and shows the result.
+     * Evaluates  math expression and shows the result.
      *
      * @param equation String that contains math expression
```
Comment (alexander-yevsyukov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java]: > Evaluates  math expression and shows the result.
Evaluates *a* math expression...

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/BinaryOperator.java [hunk 3]
```diff
@@ -20,14 +20,14 @@
 package com.teamdev.javaclasses.aleksandrov.calculator;
 
 /**
- * An interface to make operations between two numbers.
+ * An interface  to make operations between two numbers.
  *
  * @param <O>
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/BinaryOperator.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/BinaryOperatorRepository.java [hunk 4]
```diff
@@ -22,20 +22,20 @@
...
      * @return Full set of strings representing binary operator
      */
     Set<String> getRepresentations();
 
     /**
-     * Gets {@link BinaryOperator} object.
+     * Gets {@link BinaryOperator}  object.
      *
      * @param represetation String with specific representation of operator
      * @return BinaryOperator object
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/BinaryOperatorRepository.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/CalculationState.java [hunk 5]
```diff
@@ -20,25 +20,25 @@
...
 /**
- * All possible states of finite state machine during calculation.
+ * All possible  states of finite state machine during calculation.
  *
  * @author Oleg Melnik
  */
 public enum CalculationState {
     /**
-     * Initial state of machine
+     * Initial  state of machine
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/CalculationState.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/CalculationState.java [hunk 6]
```diff
@@ -20,25 +20,25 @@
...
      */
     START,
     /**
-     * State when we met a number inside expression
+     * State when  we met a number inside expression
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/CalculationState.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java (hunk 2 above)
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java [hunk 7]
```diff
@@ -53,7 +53,7 @@ public double evaluate(String equation) throws IncorrectEvaluationException {
     }
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/FSMCalculator.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/finitestatemachine/MachineRunner.java [hunk 8]
```diff
@@ -25,15 +25,15 @@
 import java.util.Set;
 
 /**
- * An abstract class that describes the logic of finite state machine.
+ * An abstract class that describes the logic  of finite state machine.
  *
  * @param <Result>        The result of machine work
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/calculator/finitestatemachine/MachineRunner.java]: Fixed

**Pull request merged**